addons/website_profile/views/website_profile.xml

```diff
@@ -511,8 +521,13 @@
                     <t t-call="website_profile.top3_user_card"></t>
                 </div>
             </div>
+            <table class="table table-sm" t-if='current_user'>
+                <tr t-foreach="current_user" t-as="user" t-attf-onclick="location.href='/profile/user/#{user['id']}';" class="o_wprofile_pointer bg-white o_wprofile_border_focus">
+                    <t t-call="website_profile.all_user_card"/>
+                </tr>
+            </table>
             <table class="table table-sm" t-if='users'>
-                <tr t-foreach="users" t-as="user" t-attf-onclick="location.href='/profile/user/#{user['id']}';" class="o_wprofile_pointer bg-white">
+                <tr t-foreach="users" t-as="user" t-attf-onclick="location.href='/profile/user/#{user['id']}';" t-attf-class="o_wprofile_pointer bg-white #{user['id'] == uid and 'o_wprofile_border_focus'}">
```

Comment:
while you touch this line 
t-as="user" is wrong practice, since user is reserved word.
and uid ==> user.id



Please rewrite this commit before a next review

Reply:
I corrected it for the two line I touched, there is a third line where it's done but I didn't modify it, should I correct that for this line too ? 
Because I don't know if it's considered as an improvement or just refactor, like when I code in old js file where some filed aren't in camel case. And so, they will be in camel case until someone has to do something on these lines.

Reply:
inside these foreach, there is a call of all_user_car template that use a lot "user". So if I rename it, I have to rename every "user" occurrence in this template too. Should I do it ?

Reply:
yes, since user is reserved words..

You can later use again user.name, and you expect to have your user, and not the last user on which your render a card